modules/runtime/nuxeo-runtime-migration/src/main/java/org/nuxeo/runtime/migration/MigrationService.java

```diff
@@ -234,4 +237,28 @@ public long getProgressTotal() {
      */
     void runStep(String id, String step);
 
+    /**
+     * @since 11.1
+     */
+    Migration getMigration(String id);
+
+    /**
+     * @since 11.1
+     */
+    List<Migration> getMigrations();
```

Comment:
Missing javadoc, and worth moving them before `MigrationStatus getStatus(String id);`

Reply:
Now we also return null when the id is not found